docs/userguide.rst

```diff
@@ -252,9 +252,29 @@ option to the ``reana-client start`` or ``reana-client run`` commands.
 
 .. code-block:: console
 
-   $ reana-client start -w workflow.1 -o target='gendata'
+   $ reana-client start -w workflow.1 -o TARGET='gendata'
    # or
-   $ reana-client run -w workflow.1 -o target='gendata'
+   $ reana-client run -w workflow.1 -o TARGET='gendata'
+
+If you want to start workflow execution from a certain step you should use
+operational option ``FROM`` with a desired step name.
+
+.. code-block:: console
+
+   $ reana-client start -w workflow.1 -o FROM='fitdata'
+   # or
+   $ reana-client run -w workflow.1 -o FROM='fitdata'
+
+If you want to restart workflow on the same workspace you should use
+``--restart`` flag of ``reana-client start``.
+
+Note that workflow restarting can be used in a combination with operational
+options ``FROM`` or ``TARGET``.
+
+.. code-block:: console
+
+   # Restarts workflow on the same workspace from step named fitdata
+   $ reana-client start -w workflow.1 -o FROM='fitdata' --restart
```

Comment:
`s/workflow\./myanalysis\./g` -- since we tend to use `myanalysis.42` elsewhere in the docs, it would be good to harmonise onto that value